src/costab.c

```diff
@@ -26,40 +26,40 @@
 **
 *V  declaration of static variables
 */
-static Obj      objRel;                 /* handle of a relator             */
-static Obj      objNums;                /* handle of parallel numbers list */
-static Obj      objTable;               /* handle of the coset table       */
-static Obj      objTable2;              /* handle of coset factor table    */
+static Obj      objRel;                 // handle of a relator
+static Obj      objNums;                // handle of parallel numbers list
+static Obj      objTable;               // handle of the coset table
+static Obj      objTable2;              // handle of coset factor table
 static Obj      objNext;                /*                                 */
 static Obj      objPrev;                /*                                 */
 static Obj      objFactor;              /*                                 */
-static Obj      objTree;                /* handle of subgroup gens tree    */
+static Obj      objTree;                // handle of subgroup gens tree
 
-static Obj      objTree1;               /* first tree component            */
-static Obj      objTree2;               /* second tree component           */
+static Obj      objTree1;               // first tree component
+static Obj      objTree2;               // second tree component
 
-static Obj      objExponent;            /* handle of subgroup order        */
-static Obj      objWordValue;           /* handle of word value            */
+static Obj      objExponent;            // handle of subgroup order
+static Obj      objWordValue;           // handle of word value
 
-static Int      treeType;               /* tree type                       */
-static Int      treeWordLength;         /* maximal tree word length        */
+static Int      treeType;               // tree type
+static Int      treeWordLength;         // maximal tree word length
 static Int      firstDef;               /*                                 */
```

Comment:
Maybe spot entirely empty comments and just remove them?

Reply:
Indeed, my script didn't look for those. Will take care of them

Reply:
Fixed all but one were in `costab.c`, one more in `integer.c`